src/King.cpp

```diff
@@ -0,0 +1,22 @@
+#include "King.h"
+
+King::King(bool isWhite)
+	:Piece(isWhite)
+{
+}
+
+bool King::isCorrectMove(const Location& source, const Location& destination) const
+{
+	int rowDiff = std::abs(destination.x - source.x);
+	int colDiff = std::abs(destination.y - source.y);
+
+	if (rowDiff == 0 && colDiff == 0)
+		return false;
+
+	return (rowDiff <= 1 && colDiff <= 1);
+}
+
+Character King::getCharacter() const

```

Comment:
Use override keyword: In the function declarations of isCorrectMove and getCharacter, use the override keyword to explicitly indicate that these functions are overriding base class functions. This can help improve code clarity and make it easier to understand the class hierarchy.